templates/c2s-lb/config.yml

```diff
@@ -1,6 +1,6 @@
 name: C2S - Load Balancer
 description: |
   Connecting to Services - Load Balancer config for staging/production
-version: v1
-category: nhsuk beta
+version: 1.0.0
+category: nhsuk beta load-balancer
 projectURL: https://github.com/nhsuk/nhsuk-rancher-templates
```

Comment:
I'm still confused about these urls - I think this lb (staging + live) should be different to traeffic lb (dev) otherwise we will have the same issue with templates missing?